### PR TITLE
Secure Mode => Identity Verification

### PR DESCRIPTION
**Pull request by irvinebroque**
refs https://github.com/intercom/intercom/issues/73016

https://docs.google.com/document/d/1oc1DYa7aVX17zY-Jua-L2dVLcQyez6uESZWK_EAz0cw/edit

### CODE REVIEW

File: bootstrap.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ public function htmlUnclosed()
     $styles = $this->getStyles();
     $app_id = WordPressEscaper::escAttr($settings['app_id']);
     $secret = WordPressEscaper::escAttr($settings['secret']);
-    $secure_mode = WordPressEscaper::escAttr($settings['secure_mode']);
```
Comment (Skaelv) [inline bootstrap.php]: This settings option refers to a data storage in Wordpress. Simply replacing the name of the value is going to lead to retrieve nil values for current customers.

File: bootstrap.php [hunk 2]
```diff
@@ -111,8 +111,8 @@ public function htmlUnclosed()
     if (isset($_GET['authenticated'])) {
       $dismissable_message = $this->dismissibleMessage('You successfully authenticated with Intercom');
     }
-    if (isset($_GET['enable_secure_mode'])) {
```
Comment (Skaelv) [inline bootstrap.php]: This coomunicates with the wordpress_oauth server which needs to be updated too: https://github.com/intercom/wordpress-oauth/pull/11 @irvinebroque

File: bootstrap.php (hunk 1 above)
Comment (Skaelv) [inline bootstrap.php]: We need a fallback on `$settings[:secure_mode]` if no value was found for `$settings[:identity_verification]`

Comment (irvinebroque) [inline bootstrap.php]: https://github.com/intercom/intercom-wordpress/pull/80/commits/ee16ea8889ee7237b4eae04d7206fa8495da3965

File: bootstrap.php [hunk 3]
```diff
@@ -97,7 +97,7 @@ public function htmlUnclosed()
     $styles = $this->getStyles();
     $app_id = WordPressEscaper::escAttr($settings['app_id']);
     $secret = WordPressEscaper::escAttr($settings['secret']);
-    $identity_verification = WordPressEscaper::escAttr($settings['identity_verification']);
+    $identity_verification = WordPressEscaper::escAttr($settings['identity_verification'] ? $settings['identity_verification'] : $settings['secure_mode']);
```
Comment (Skaelv) [inline bootstrap.php]: Should we do ?
``` 
 $identity_verification = isset($settings['identity_verification']) ? WordPressEscaper::escAttr(($settings['identity_verification']) : WordPressEscaper::escAttr(($settings['secure_mode']));
```

Comment (Skaelv) [inline bootstrap.php]: So we don't skip the `wordpressEscaper::EscAttr` method.

Comment (irvinebroque) [inline bootstrap.php]: 👍

Comment (irvinebroque) [inline bootstrap.php]: Updated

**Comment by Skaelv** [comment]
I just approved. You can go ahead and merge and we will take care of releasing the new version to wordpress.

**Pull request merged**